examples/memory.cc

```diff
@@ -19,7 +19,7 @@ int main() {
   // Create our `store` context and then compile a module and create an
   // instance from the compiled module all in one go.
   Engine engine;
-  Module module = Module::compile(engine, readFile("examples/memory.wat")).unwrap();
+  Module module = Module::compile(engine, std::string_view(readFile("examples/memory.wat"))).unwrap();
```

Comment:
Could a new constructor be added for `Module::compile` to keep this working? Perhaps one that explicitly takes a `string_view`?

Reply:
Removed, not needed anymore